src/hotspot/share/prims/jvm.cpp

```diff
@@ -2160,6 +2176,7 @@ JVM_ENTRY(jobjectArray, JVM_GetPermittedSubclasses(JNIEnv* env, jclass current))
     }
     return (jobjectArray)JNIHandles::make_local(THREAD, result());
   } else {
+    log_trace(class, sealed)("Type %s is not sealed", ik->external_name());
```

Comment:
I would remove this line in favor of adding adding an indication of sealed or not sealed to line #2129.  See my suggested change above.